packages/genesys-spark-components/src/components/beta/gux-label-info/gux-label-info.tsx

```diff
@@ -0,0 +1,61 @@
+import { Component, Element, h, JSX, Prop, Method } from '@stencil/core';
+
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxLabelInfoVariant } from './gux-label-info.types';
+
+/**
+ * @slot content
+ */
+
+@Component({
+  tag: 'gux-label-info-beta',
+  shadow: false // shadow is false to allow gux-icon-tooltip-beta to access text for tooltip
```

Comment:
I wonder if there's a way we can access the text for the tooltip without setting shadow to false here. I'll try something on my end to see if I can think of a way.

Reply:
update to have shadow true